File: solutions/1528-kids-with-the-greatest-number-of-candies/solution.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        int len = candies.length;
        List<Boolean> result = new ArrayList<>();

        for(int i=0;i<len;i++){
            int sum = candies[i] + extraCandies;
            boolean great = true;
            for(int j=0;j<len;j++){
                if(sum < candies[j]){
                    great = false;
                }
            }
            result.add(great);
        }

        return result;
    }
}

File: solutions/1528-kids-with-the-greatest-number-of-candies/solution.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        larges = max(candies)
        for x in range(len(candies)):
            if candies[x] + extraCandies >= larges:
                candies[x] = True
            else:
                candies[x] = False
        return candies

